playwright/src/test/java/com/microsoft/playwright/Utils.java

```diff
@@ -122,4 +122,11 @@ static BrowserType getBrowserTypeFromEnv(Playwright playwright) {
         throw new IllegalArgumentException("Unknown browser: " + browserName);
     }
   }
+
+  static void verifyViewport(Page page, int width, int height) {
```

Comment:
Can you update TestBrowserContextBasic to use this method and get rid of its local one?

Reply:
Will do